src/main/java/mod/chiselsandbits/items/ItemNegativePrint.java

```diff
@@ -151,6 +151,22 @@ public EnumActionResult onItemUse(
 		final ItemStack stack = player.getHeldItem( hand );
 		final IBlockState blkstate = world.getBlockState( pos );
 
+		if ( ChiselsAndBits.getConfig().requireBagSpace && !player.isCreative() )
+		{
+			//Cycle every item in any bag, if the player can't store the clicked block then
+			//send them a message.
+			final int stateId = ModUtil.getStateId( blkstate );
+			if ( !ItemChiseledBit.hasBitSpace( player, stateId ) )
+			{
+				if( player.worldObj.isRemote )
+				{
+					//Only client should handle messaging.
+					player.addChatMessage( new TextComponentTranslation( "mod.chiselsandbits.result.require_bag" ) );
+				}
+				return EnumActionResult.FAIL;
+			}
+		}
+
```

Comment:
This, is more or less the same block of code that is in the positive print, is it the same as another place as well? Would it be better to convert this into a common method? if ( ItemChiseledBit.checkRequiredSpace( player, blkstate ) ) return EnumActionResult.FAIL; or something along those lines?